migrate-this/credits_controller.js

```diff
@@ -11,6 +14,46 @@ function notify_button_exists(button) {
   }
 }
 
+
+
 (function() {
   notify_button_exists('[data-uia="watch-credits-seamless-button"]');
-})();
\ No newline at end of file
+})();
+
+
+
+
+PlayerControlsNeo__all-controls
+PlayerControlsNeo__all-controls
+
+PlayerControlsNeo__layout
+
+*/
+
+setTimeout(function() {
```

Comment:
Why would you use a timeout function instead of just wrapping this part of the code inside a try block with a timer?